docs/reference/dapis/understand/index.md

```diff
@@ -128,6 +108,24 @@ This is advantages as the dApp owner does not need to worry about the community
 based funding model that might cause the dAPI to shut down due to lack of
 funding.
 
-<FlexEndTag/>
+## Availability
+
+Both **Self-funded dAPIs** and **Managed dAPIs** are available on the
+[Market](https://market.api3.org/dapis).
+
+| Self-funded dAPIs                                                               | Managed dAPIs                                         |
+| ------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| March 2023                                                                      | Fall 2023                                             |
+| Single public [proxy contract](/reference/dapis/understand/proxy-contracts.md)  | Single public proxy contract                          |
+| 1% deviation                                                                    | Multiple deviations<br/>(0.25%, 0.5%, 1%)             |
+| 60 second [interval](/reference/dapis/understand/deviations.md#update-interval) | 30-60 second interval                                 |
+| 24 hour [heartbeat](/reference/dapis/understand/deviations.md#heartbeat)        | 2 minute or 24 hour heartbeat                         |
+| Sourced from a single<br/>data feed (beacon)                                    | Sourced from multiple<br/>data feeds (beacons)        |
+| Gas costs are community funded                                                  | Gas costs are managed <br/>by API3 using upgrade fees |
```

Comment:
I feel like this table needs a leftmost row descriptor column e.g. for the first row "Available since", second row "Structure", third row "Deviation", etc.